### ISSUE TITLE
Add support for PS Now

### ISSUE THREAD

**Issue by Wombarly** [FriendsOfGalaxy/galaxy-integration-psn#9]
Using PS Now it's possible to play certain PlayStation games on your PC. So being able to launch them through GOG Galaxy would be nice.

They require the PS Now launcher, not sure if there is a way to directly launch titles with some kind of URI though.

**Comment by FriendsOfGalaxy** [comment]
That is interesting idea. If anyone want to research technical possibilities, we'll appreciate

**Comment by ShaunV2** [comment]
I don't think this idea will work at all, for one I don't think it displays in the window title what game you're playing and it's not possible to make shortcuts to the games you're streaming unlike on the actual PS4 so launching by URL is probably not possible, this idea is just wishful thinking.

I would only use the PS Now Launcher on PC to stream PS3 games (feels like it does it better than the actual PS4) and nothing else, having an actual PS4 is best for the PS2 and PS4 games since you can download them to the PS4.

**Comment by adamrb** [comment]
As far as I can tell, this is the (US EN) API endpoint to list PS Now games: https://browse.prod.gaikai.com/apollo1/v1/np/us/en

Looks like it should be pretty simple to integration with the new "Subscriptions" feature of the Galaxy API: https://galaxy-integrations-python-api.readthedocs.io/en/latest/galaxy.api.html?highlight=subscription#galaxy.api.types.Subscription
